File: app/src/main/java/social/entourage/android/api/LoginRequest.java
package social.entourage.android.api;

import java.util.HashMap;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LoginRequest {

    @POST("login.json")
    Call<LoginResponse> login(@Body HashMap<String, String> user);
}

File: app/src/main/java/social/entourage/android/api/LoginRequest.kt
package social.entourage.android.api

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST
import social.entourage.android.api.model.User
import java.util.*


class LoginResponse(val user: User)

interface LoginRequest {
    @POST("login.json")
    fun login(@Body user: HashMap<String, String>): Call<LoginResponse>
}